plugins/typescript/src/generators/generateReactQueryComponents.ts

```diff
@@ -65,7 +65,12 @@ export const generateReactQueryComponents = async (
       .join("\n");
 
   const filenamePrefix =
-    c.snake(config.filenamePrefix || context.openAPIDocument.info.title) + "-";
+    c.snake(
+      isNull(config.filenamePrefix)
+        ? ""
+        : config.filenamePrefix || context.openAPIDocument.info.title
```

Comment:
Now that you point at this issue, I think we should use `??` instead `||` so we can setup `filePrefix` to `""` or `null`

Reply:
Oh yeah, although it wouldn't be ``null`` just empty string

Reply:
PR updated, makes more sense now